src/components/BreedsList.vue

```diff
@@ -68,16 +68,15 @@ import NavigationButton from "./NavigationButton.vue";
 import BreedDetails from "./BreedDetails.vue";
 
 import TheNavigation from "./TheNavigation.vue";
-import store from "../store";
-
+import { useState } from "../store.js";
+const useGlobalState = useState();
```

Comment:
zakaj si to naredu zunej Vue APIja / instance?

Reply:
ubistvu, nism sploh pomislu,da bi jo kam drugam dal ;)

Reply:
v Vue, zato ga maš :D v dato, computed itd.. odvisno kaj rabiš.. je pa res da je to mal čudn primer in sploh ne optimalen, ker se učiva ubistvu in nisem čist sigurn če bi sploh delal...

zato obstaja `Pinia` da je vse optimizirano in ready za uporabo z Vue. 
Problem s tem da imaš svoj store je v tem ker nima raznih "bombončkov" k jih Vue rab da pol dela reactivity proper, recimo kasnej k bova pogledala Vue 3 se lahko še tega dotakneva. Primer: ne moreš ti samo zawrepat `state.name` v computed in ko se v store spremeni `name`, bi mogu ta computed reagirat.. pa ne :D 

kva misleš, greš kr Pinio implementirat, da se ne ubijaš preveč z raznimi hacki da bi to delal?

Reply:
Sm opazu ja, da ne dela ta reactivity ja 😃 
Sm za ja, bi sov kr na Pinio...kr zaprem ta PR?